pkg/builddef/healthcheck.go

```diff
@@ -0,0 +1,186 @@
+package builddef
```

Comment:
Shouldn't this file also be covered by an unit test ? It would describe its use cases.

Reply:
I should admit that I'm a bit lazy :stuck_out_tongue:  All these funcs/methods are already tested via defkind handlers, so I think it'd be quite redundant to test them separately for now. However if there's any bug or if any refactoring is needed to improve this file, I'll add tests :wink: